### PR TITLE
Adds dirty info for lexx_brahmin-variant.esp and lexx_brahmin-betsy.esp

### PR DESCRIPTION
**Pull request by abiel0530**

### CODE REVIEW

**Comment by Ortham** [comment]
CI is failing because you've got the wrong indentation for some fields. E.g.

```
  - name: 'lexx_brahmin-variant.esp'
    dirty:
      - crc: 0xB596767C
      << *dirtyPlugin
      itm: 16
      udr: 1
```

should be 

```
  - name: 'lexx_brahmin-variant.esp'
    dirty:
      - crc: 0xB596767C
        << *dirtyPlugin
        itm: 16
        udr: 1
```

**Comment by abiel0530** [comment]
Whoops, sorry about that, fixed it. Please squash the earlier commit.

File: masterlist.yaml [hunk 1]
```diff
@@ -5325,3 +5325,15 @@ plugins:
         <<: *dirtyPlugin
         itm: 1
         udr: 0
+  - name: 'lexx_brahmin-variant.esp'
+    dirty:
+      - crc: 0xB596767C
+        << *dirtyPlugin
```
Comment (Ortham) [inline masterlist.yaml]: This needs to be `<<: *dirtyPlugin`, same with line 5337.

**Review by Ortham — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
@abiel0530 you can do rebase squash localy and force push yourself

**Comment by Ortham** [comment]
> @abiel0530 you can do rebase squash localy and force push yourself

It's trivial to squash and merge a PR through GitHub though, and it avoids another round of CI. You can even resolve merge conflicts through the web UI these days, though I find it slightly confusing.

**Comment by abiel0530** [comment]
@WrinklyNinja uh, did they squash together nicely? Git says so, after doing the `git rebase` thing

**Pull request merged**